src/auspex/analysis/qubit_fits.py

```diff
@@ -441,6 +441,53 @@ def get_pops(self):
         pop2_err = np.std(np.reshape(self.pop2,(len(self.lengths),repeats)),1)
         return pop0, pop0_err, pop1, pop1_err, pop2, pop2_err
 
+def InterleavedError(base_fit: SingleQubitRBFit, 
```

Comment:
👍  we should be better about typing generally...

Reply:
I think python > 3.5 supports this kind of type hinting. No reason to not start making this a habit. It also make the code more understandable IMO.